src/core/CoreTransaction.jl

```diff
@@ -78,6 +93,7 @@ function close(transaction::T)::Bool where {T<:AbstractCoreTransaction}
         delete!(transaction.session, transaction.transaction_id)
     catch ex
         throw(TypeDBClientException("something went wrong closing Transaction", ex))
+        return false
```

Comment:
This line would never be reached since it throws right above...

Reply:
Done.